snippets/graphql/create-component/README.md

```diff
@@ -0,0 +1,45 @@
+# How to use this
+
+This query will retrieve a list of metric definitions for your site.
+
+Replace `cloudId` below in the variables section with the cloudId for your site and execute the query. You can use [the GraphQL explorer](https://developer.atlassian.com/cloud/compass/graphql/explorer/) to run this query and explore [the Compass API](https://developer.atlassian.com/cloud/compass/graphql/) further.
+
+### Query
+
+```graphql
+mutation createComponent($cloudId: ID!, $componentDetails: CreateCompassComponentInput!) {
+  compass {
+    createComponent(cloudId: $cloudId, input: $componentDetails) {
+      success
+      componentDetails {
+        id
+        name
+        type
```

Comment:
When I run this query, it marks `type` as deprecated in favor of `typeId`. The query still works, though.

Reply:
good point, i should translate this back to typeID, we introduced that recently.